src/Combinatorics/Graphs/functions.jl

```diff
@@ -1097,37 +1097,36 @@ function graph_from_edges(::Type{T},
   return g
 end
 
-graph_from_edges(::Type{T},
-edges::EdgeIterator,
-n_vertices::Int=-1) where {T <: Union{Directed, Undirected}} = graph_from_edges(T, collect(edges), n_vertices)
+function graph_from_edges(::Type{T},
+                          edges::EdgeIterator,
+                          n_vertices::Int=-1) where {T <: Union{Directed, Undirected}}
+  return graph_from_edges(T, collect(edges), n_vertices)
+end
 
 @doc raw"""
-  graph_from_edges(::Type{T}, edges::Vector{Vector{Int}}) where {T <:Union{Directed, Undirected}}
-
-Creates a graph from a vector of edges. Optionally, you could input the number of vertices, but if this number is lower than the maximum vertex in the edges, this argument will be ignored.
+    graph_from_edges(edges::Vector{Vector{Int}})
+    graph_from_edges(::Type{T}, edges::Vector{Vector{Int}}, n_vertices::Int=-1) where {T <:Union{Directed, Undirected}}
+Creates a graph from a vector of edges. There is an optional input for number of vertices, `graph_from_edges`  will
```

Comment:
```suggestion
    graph_from_edges(edges::Vector{Vector{Int}})
    graph_from_edges(::Type{T}, edges::Vector{Vector{Int}}, n_vertices::Int=-1) where {T <:Union{Directed, Undirected}}

Creates a graph from a vector of edges. There is an optional input for number of vertices, `graph_from_edges`  will
```
empty line between signature and text